metaflow/exception.py

```diff
@@ -91,6 +91,13 @@ def __init__(self, namespace):
         super(MetaflowNamespaceMismatch, self).__init__(msg)
 
 
+class MetaflowInvalidPathspec(MetaflowException):
+    headline = "Invalid pathspec for the MetaflowObject type"
```

Comment:
Can we tweak this to just read - `Invalid pathspec`? Otherwise the PR is ready to be shipped.